data/mods/Magiclysm/Spells/attunements/Void_Mage.json

```diff
@@ -0,0 +1,164 @@
+[
+  {
+    "type": "SPELL",
+    "id": "voidmage_final_grasp",
+    "name": "Final Grasp",
+    "description": "In the end, everything succumbs to the grasp of nonexistence and stasis.  This spell gives the target a foretaste of that.",
+    "effect": "attack",
+    "shape": "blast",
+    "valid_targets": [ "hostile" ],
+    "flags": [ "SILENT", "SOMATIC", "IGNORE_WALLS", "NO_LEGS" ],
+    "min_damage": 0,
+    "max_damage": 0,
+    "min_range": 5,
+    "max_range": 25,
+    "range_increment": 0.7,
+    "damage_type": "pure",
+    "min_dot": 5,
+    "max_dot": 55,
+    "dot_increment": 1.5,
+    "max_level": 35,
+    "min_duration": 500,
+    "max_duration": 5000,
+    "duration_increment": 130,
+    "spell_class": "VOID_MAGE",
+    "base_casting_time": 200,
+    "energy_source": "MANA",
+    "base_energy_cost": 300,
+    "final_energy_cost": 150,
+    "energy_increment": -5,
+    "difficulty": 9,
+    "extra_effects": [ { "id": "voidmage_final_grasp_slowdown" } ]
+  },
+  {
+    "id": "voidmage_final_grasp_slowdown",
+    "type": "SPELL",
+    "name": "Final Grasp slowdown",
+    "description": "Removes some moves from an enemy.  Subspell; you aren't allowed to see it except in debug mode.",
+    "valid_targets": [ "hostile" ],
+    "effect": "mod_moves",
+    "shape": "blast",
+    "max_level": 35,
+    "min_damage": -100,
+    "max_damage": -3000,
```

Comment:
30 seconds of a completely frozen enemy seems a bit extreme, doesn't it?

Reply:
for an attunement spell, on max level (its a bit lesser than 20 second on 20 lvl), i dont think so, but i can reduce it if its necessary